AnnoyancesFilter/sections/cookies_general.txt

```diff
@@ -883,6 +883,7 @@
 ###survey_bar
 ###svtCookieInformationContainer
 ###tarteaucitronAlertBig
+###tarteaucitronRoot
```

Comment:
It seems this related to the `###tarteaucitronAlertBig`. Usually we add a rule with domain, but in this case I guess it's ok.